Tests/SwiftCliCoreTests/InputValidatorTest.swift

```diff
@@ -7,24 +7,49 @@ class ValidatorTests: XCTestCase {
     func testInvalidStitchShouldThrowError() throws {
```

Comment:
Look into using `setUp()` to minimize redundant test setup in this `XCTestCase` instance. https://developer.apple.com/documentation/xctest/xctestcase/1496262-setup

Reply:
Done!